src/Diagnostics.RuntimeHost/Models/PostBodyModel.cs

```diff
@@ -12,6 +12,7 @@ public class DiagnosticSiteData
         public string NamespaceDescriptor { get; set; }
         public bool IsDefaultContainer { get; set; }
         public bool? IsLinux { get; set; }
+        public bool IsXenon => Kind != null && Kind.ToLower().Contains("xenon");
```

Comment:
use `string.IsNullOrWhiteSpace` instead of `Kind != null`

Reply:
I felt that this form was more succinct given `Contains` will return `false` on a whitespace/empty string